known-shortcuts/numbers.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
```

Comment:
:bug: **Bug**
The `app` object is being cast to `AppParams` using the `as` keyword. While this is a valid TypeScript pattern, it is generally safer to use type annotations directly in the object definition to ensure that the object conforms to the `AppParams` type. This can help catch any potential type mismatches at compile time. Additionally, the `AppParams` type is imported from `"~/types"`, but there is no indication in the provided context that this type is defined or what it includes. If `AppParams` is not defined or does not match the structure of the `app` object, this could lead to a TypeScript error. Ensure that `AppParams` is correctly defined and matches the structure of the `app` object.

```suggestion
// Ensure AppParams is defined in ~/types and matches the app structure
```